From 275671c1b51271b0252e7c696515812fe0020908 Mon Sep 17 00:00:00 2001
From: Florian Klink <flokli@flokli.de>
Date: Sun, 29 Mar 2020 03:02:15 +0200
Subject: [PATCH] xilinx: dma: fix some typos in the Kconfig help texts

Signed-off-by: Florian Klink <flokli@flokli.de>
---
 drivers/gpu/drm/xlnx/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/Kconfig b/drivers/gpu/drm/xlnx/Kconfig
index ecc5f90caf61..5463d4aa0302 100644
--- a/drivers/gpu/drm/xlnx/Kconfig
+++ b/drivers/gpu/drm/xlnx/Kconfig
@@ -41,7 +41,7 @@ config DRM_ZYNQMP_DPSUB
 	  DRM KMS driver for ZynqMP DP Subsystem controller. Choose
 	  this option if you have a Xilinx ZynqMP SoC with DisplayPort
 	  subsystem. The driver provides the kernel mode setting
-	  functionlaities for ZynqMP DP subsystem.
+	  functionalities for ZynqMP DP subsystem.
 
 config DRM_XLNX_DSI
 	tristate "Xilinx DRM DSI Subsystem Driver"
@@ -59,7 +59,7 @@ config DRM_XLNX_MIXER
 	depends on DRM_XLNX
 	select VIDEOMODE_HELPERS
 	help
-	  DRM driver for Xilinx Mixer driver
+	  DRM driver for Xilinx Mixer driver.
 
 config DRM_XLNX_PL_DISP
 	tristate "Xilinx DRM PL display driver"
-- 
2.25.1

